MAUI/Scheduler/agenda-view.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: post
+title: Agenda view in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about the Agenda view feature of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Agenda view in .NET MAUI Scheduler (SfScheduler)
+
+The agenda view displays the events or appointments (normal appointments, all-day appointments, recurring appointments, and span appointments) in chronological order, grouped by date between the minimum and maximum dates and the current date displayed by default. When the `AppointmentsSource` property of `SfScheduler` is `null,` the agenda view will show only the month, week, and date headers.
+
+A agenda view displays different UI for mobile and desktop, for mobile it displays the month header, the week header, and the date header however for desktop, it displays the appointment only.
+
+N> When the desktop view width is less than 600, the scheduler will display the mobile agenda UI on the desktop.
+
+{% tabs %}
+{% highlight xaml %}
+
+ <scheduler:SfScheduler x:Name="Scheduler" 
+                        View="Agenda">
+ </scheduler:SfScheduler>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.Scheduler.View = SchedulerView.Agenda;
+// Creating an instance for the scheduler appointment collection.
+var appointments = new ObservableCollection<SchedulerAppointment>();
+// Adding scheduler appointment in the scheduler appointment collection.
+appointments.Add(new SchedulerAppointment()
+{
+    Subject = "Meeting",
+    StartTime = DateTime.Now,
+    EndTime = DateTime.Now.AddHours(1),
+    Background = Brush.Orange,
+});
+// Adding scheduler appointment into the AppointmentsSource.
+this.Scheduler.AppointmentsSource = appointments;
+
+{% endhighlight %}
+{% endtabs %}
```

Comment:
Change the order of the topics

1. Month Header,
2. Week Header
3. Day Header
4. Appointment text customization

Reply:
Re-ordered the topics.